### PR TITLE
Remove unnecessary line breaks + Cleanup YML-files

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -34,4 +34,3 @@
 # Avoid merge conflicts in CHANGELOG
 # https://about.gitlab.com/2015/02/10/gitlab-reduced-merge-conflicts-by-90-percent-with-changelog-placeholders/
 /CHANGELOG.md		merge=union
-
```
Comment (xepozz) [inline .gitattributes:37]: It is a good practice to leave empty lines in the end of file.
Why do you think vice versa?

**Review by xepozz — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (vjik) [reply .gitattributes:37]: Now two empty lines in the end of this file. I'm leave only one empty line.

Comment (xepozz) [reply .gitattributes:37]: As I see you removed it.
image

Comment (vjik) [reply .gitattributes:37]: GitHub don't show last empty line. See any PHP file. For example: https://github.com/yiisoft/html/blob/master/src/NoEncodeStringableInterface.php

When file don't have last empty line github show icon:
image

**Review by samdark — APPROVED** [review]

**Pull request merged**